core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -234,14 +237,16 @@ private void transitionRequestAck(String processId) {
         transferProcessStore.update(transferProcess);
     }
 
+    @WithSpan("initiate_request_transfer_process")
```

Comment:
"initiate_transfer_process_request"? Its more in line with the naming for the contract negotiation.